cmd/vulcan-trivy/vulcan-trivy.go

```diff
@@ -186,116 +194,108 @@ func run(ctx context.Context, target, assetType, optJSON string, state checkstat
-		vResourcesTable["Name"] = op.name
-		vResourcesTable["Version"] = op.version
-		vResourcesTable["FixedBy"] = op.fixedBy
-
-		for i := 0; i < len(op.cves) && i < vulnCVETrucateLimit; i++ {
-			vResourcesTable["Vulnerabilities"] = fmt.Sprintf("%s | [%s](https://nvd.nist.gov/vuln/detail/%s)", vResourcesTable["Vulnerabilities"], op.cves[i], op.cves[i])
-		}
-		if len(op.cves) > vulnCVETrucateLimit {
-			logger.Warnf("truncate affected package [%s] CVE list to [%d]\n", op.name, vulnCVETrucateLimit)
-			vResourcesTable["Vulnerabilities"] = fmt.Sprintf("%s | and some others ...)", vResourcesTable["Vulnerabilities"])
+		vp := report.ResourcesGroup{
+			Name: "Package Vulnerabilities",
+			Header: []string{
+				"FixedBy",
+				"CVE",
+				"Severity",
+				"CWEs",
+				"Link",
+			},
 		}
+		for key, l := range outdatedPackageVulns {
+			vp.Rows = []map[string]string{}
+			maxScore := getScore("NONE")
+			for i, l := range l {
+				if i > vulnCVETrucateLimit {
+					break
+				}
+				newScore := getScore(l.severity)
+				if newScore > maxScore {
+					maxScore = newScore
+				}
+				row := make(map[string]string, len(vp.Header))
+				row["FixedBy"] = l.fixedBy
+				if l.cves != nil {
+					row["CWEs"] = strings.Join(l.cves, ",")
+				}
+				row["CVE"] = l.cve
+				if l.link == "" {
+					l.link = fmt.Sprintf("https://nvd.nist.gov/vuln/detail/%s", l.cve)
+				}
+				row["Link"] = l.link
+				row["Severity"] = l.severity
+				vp.Rows = append(vp.Rows, row)
+			}
 
-		vp.Rows = []map[string]string{vResourcesTable}
+			// Build the vulnerability.
+			vuln := report.Vulnerability{
+				// Issue attributes.
+				Summary:     "Outdated Packages in Docker Image",
+				Description: "Vulnerabilities have been found in outdated packages installed in the Docker image.",
+				Recommendations: []string{
+					"Update affected packages to the versions specified in the resources table or newer.",
+				},
+				CWEID:  937, // TODO: Should we keep this or compute from the from CWES? What if more than one?
+				Labels: []string{"potential", "docker"},
+				// Finding attributes.
+				Score:     maxScore,
+				Details:   generateDetails(registryEnvDomain, target),
+				Resources: []report.ResourcesGroup{vp},
+			}
 
-		// Build the vulnerability.
-		vuln := report.Vulnerability{
-			// Issue attributes.
-			Summary:     "Outdated Packages in Docker Image",
-			Description: "Vulnerabilities have been found in outdated packages installed in the Docker image.",
-			Recommendations: []string{
-				"Update affected packages to the versions specified in the resources table or newer.",
-			},
-			CWEID:  937,
-			Labels: []string{"potential", "docker"},
-
-			// Finding attributes.
-			Fingerprint:      fingerprint,
-			AffectedResource: affectedResource,
-			Score:            getScore(op.severity),
-			Details:          generateDetails(registryEnvDomain, target),
-			Resources:        []report.ResourcesGroup{vp},
-		}
+			vuln.Fingerprint = helpers.ComputeFingerprint()
```

Comment:
What would be the fingerprint?

Reply:
The fingerprint for this vulnerability should be an array formed by:
The list of CVEs (not CWEs) + the list of severities of each CVE.
All sorted.